src/Export/Field/FilterAttributes.php

```diff
@@ -39,7 +39,11 @@ protected function getVariantValues(Article $article, Article $parent): string
             return $this->filter->filterValue($key) . '=' . $this->filter->filterValue($value) . '|';
         }, ...array_map(function (string $value): array {
             return explode(' | ', $value);
-        }, [$parent->getFieldData('oxvarname'), $article->getFieldData('oxvarselect')])));
+        }, [
+                $parent->getFieldData('oxvarname') ? $parent->getFieldData('oxvarname') : 'missing field data',
```

Comment:
I'm not sure, should we return some information message or just empty string if getFieldData returns `null`

@a-laurowski

Reply:
i'd go with empty string, any other value will be evaluated by the FACT-Finder. We rather don't want to have a filter with option "missing field data" :D